dev-tools/mage/gotest.go

```diff
@@ -83,13 +84,18 @@ func makeGoTestArgsForModule(name, module string) GoTestArgs {
 		Packages:        []string{fmt.Sprintf("./module/%s/...", module)},
 		OutputFile:      fileName + ".out",
 		JUnitReportFile: fileName + ".xml",
+		Tags:            testTagsFromEnv(),
 	}
 	if TestCoverage {
 		params.CoverageProfileFile = fileName + ".cov"
 	}
 	return params
 }
 
+func testTagsFromEnv() []string {
```

Comment:
maybe adding an example here for what TEST_TAGS should look like will help us when we try to run it locally.

Reply:
I have added some comments in the code about this. In principle same comma-separated format can be used as with the go `--tags` flag.

Reply:
Thank you!!

Reply:
Thanks!